test/code/plugins/VMInsightsDataCollector_subtest.rb

```diff
@@ -376,14 +376,15 @@ def test_get_filesystems_live
                 data = io.readlines
                 data.each { |line|
                     s = line.split(" ")
-                    expected[s[0]] = [ s[1], s[2].to_i, false ]
+                    ind = s[0].index('/', 1)
+                    key = s[0][ind+1..-1]
```

Comment:
Suggestion: The code, searching for the / and using its index to remove the prefix seems to be overly general, and introduces unhandled error paths. Simplify. We know the string must start with "/dev/", and that is what we need to remove.

Reply:
I agree with this. Have made changes to ensure uniformity of operation. Since in Linux all devices are located in /dev/ it has been made a necessary check.